.travis.yml

```diff
@@ -54,14 +54,11 @@ jobs:
           -- -- -j2
         - cd $BASEPATH/tests
         - python -m pytest
-        - cd $BASEPATH/examples/jupyter
-        - jupyter nbconvert --execute --ExecutePreprocessor.timeout=None *.ipynb
         - cd $BASEPATH/doc
         - make linkcheck
         - make html
       before_deploy:
         - cd $BASEPATH
-        - rm -rf doc/build/html/_sources
```

Comment:
How do you make sure that the source is not deployed to the gh pages repo?

Reply:
See [here](https://github.com/ComPWA/pycompwa/pull/74/commits/80635976fa8c55e8bffe9f241f51c3d896d4b3ea#diff-555c472a89566734e0d8567fb205d5dfR167)